server/src/main/java/io/druid/server/http/LookupCoordinatorResource.java

```diff
@@ -284,4 +295,346 @@ public Response getSpecificTier(
       return Response.serverError().entity(ServletResourceUtils.sanitizeException(e)).build();
     }
   }
+
+  @GET
+  @Produces({MediaType.APPLICATION_JSON})
+  @Path("/status")
+  public Response getAllLookupsStatus(
+      @QueryParam("detailed") boolean detailed
+  )
+  {
+    try {
+      Map<String, Map<String, LookupExtractorFactoryMapContainer>> configuredLookups = lookupCoordinatorManager.getKnownLookups();
+      if (configuredLookups == null) {
+        return Response.status(Response.Status.NOT_FOUND)
+                       .entity("No lookups found")
```

Comment:
well on client side input stream will return characters `No lookups found` , which is definitely not parseable into json. It can be made json by sending `{"error": "No lookups found"}` instead.
That said, do we really need to return valid json on Non-200 error response?

BTW, in most cases these null checks are not required because they are PathParams and can never be null or empty. removed those.